### PR TITLE
Removed unecessary default

### PR DESCRIPTION
**Pull request by hackaugusto**
https://github.com/raiden-network/raiden/pull/3218#discussion_r245319411

### CODE REVIEW

**Comment by LefterisJP** [comment]
Aha ... that's why I did it! I remember now.

I had rebased on master and had some problems with yoichi's changes: 

Same reason your tests failed right now.

```
raiden/network/proxies/token_network.py:266: error: Argument 3 to "_get_channel_state" of "TokenNetwork" has incompatible type "Optional[ChannelID]"; expected "ChannelID"
raiden/network/proxies/token_network.py:409: error: Argument 3 to "_get_channel_state" of "TokenNetwork" has incompatible type "Optional[ChannelID]"; expected "ChannelID"
raiden/network/proxies/token_network.py:422: error: Argument 3 to "_get_channel_state" of "TokenNetwork" has incompatible type "Optional[ChannelID]"; expected "ChannelID"
raiden/network/proxies/token_network.py:435: error: Argument 3 to "_get_channel_state" of "TokenNetwork" has incompatible type "Optional[ChannelID]"; expected "ChannelID"
```

**Comment by hackaugusto** [comment]
@LefterisJP please have a second look

File: raiden/tests/integration/contracts/test_token_network.py [hunk 1]
```diff
@@ -102,8 +102,22 @@ def test_token_network_proxy_basics(
         c1_client.address,
         c2_client.address,
     ) is False
-    assert c1_token_network_proxy.channel_is_opened(c1_client.address, c2_client.address) is False
-    assert c1_token_network_proxy.channel_is_closed(c1_client.address, c2_client.address) is False
+
+    channel_identifier = c1_token_network_proxy._call_and_check_result(
```
Comment (LefterisJP) [inline raiden/tests/integration/contracts/test_token_network.py]: tricky :)

Comment (hackaugusto) [reply raiden/tests/integration/contracts/test_token_network.py]: yeah, this was lazyness on my part :P

**Pull request merged**